packages/appstore/src/components/wallets-banner/wallets-banner.tsx

```diff
@@ -1,94 +1,24 @@
 import React from 'react';
-import { TWalletsMigrationStatus } from 'Types';
 import WalletsBannerUpgrade from './wallets-banner-upgrade';
 import WalletsBannerUpgrading from './wallets-banner-upgrading';
 import WalletsBannerReady from './wallets-banner-ready';
+import { useWalletMigration } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
 
-// just for testing purpose and will be deleted in the future
-type TBannerSwitcher = {
-    status: TWalletsMigrationStatus;
-    onChangeStatus: (value: TWalletsMigrationStatus) => void;
-    is_eu: boolean;
-    onChangeEU: (value: boolean) => void;
-    children?: React.ReactNode;
-};
+const WalletsBanner = observer(() => {
+    const { is_eligible, is_failed, is_in_progress, is_migrated, is_ineligible } = useWalletMigration();
+    const { traders_hub } = useStore();
+    const { is_eu_user } = traders_hub;
 
-// just for testing
-const BannerSwitcher = ({ status, is_eu, onChangeStatus, onChangeEU, children }: TBannerSwitcher) => {
-    const onStatusChangeHandler = (event: React.ChangeEvent<HTMLSelectElement>) => {
-        onChangeStatus(event.target.value as TWalletsMigrationStatus);
-    };
+    if (is_ineligible) return null;
 
-    const onEUChangeHandler = (event: React.ChangeEvent<HTMLSelectElement>) => {
-        if (event.target.value === 'EU') onChangeEU(true);
-        else if (event.target.value === 'Non-EU') onChangeEU(false);
-    };
+    if (is_eligible || is_failed) return <WalletsBannerUpgrade />;
 
-    return (
-        <React.Fragment>
-            <div>
-                <select name='status' onChange={onStatusChangeHandler} value={status}>
-                    <option value='ineligible'>ineligible</option>
-                    <option value='eligible'>eligible</option>
-                    <option value='in_progress'>in_progress</option>
-                    <option value='migrated'>migrated</option>
-                    <option value='failed'>failed</option>
-                </select>
-                <select name='is_eu' onChange={onEUChangeHandler} value={is_eu ? 'EU' : 'Non-EU'}>
-                    <option value='EU'>EU</option>
-                    <option value='Non-EU'>Non-EU</option>
-                </select>
-            </div>
-            {children}
-        </React.Fragment>
-    );
-};
+    if (is_in_progress) return <WalletsBannerUpgrading is_eu={is_eu_user} />;
 
-const WalletsBanner = () => {
-    // just for testing purpose
-    const [migration_status, setMigrationStatus] = React.useState<TWalletsMigrationStatus>('eligible');
-    const [is_eu, setIsEu] = React.useState(false);
+    if (is_migrated) return <WalletsBannerReady is_eu={is_eu_user} />;
```

Comment:
@thisyahlen-deriv I think you can delete this string because next string return the same component and we don't need to check is it `is_migrated` or no